docs/reference/commandline/run.md

```diff
@@ -874,6 +874,17 @@ $ docker run --rm -i busybox echo "foo bar baz" \
 rab
 ```
 
+## <a name="init"></a> Specify an init process
+
+You can use the `--init` flag to indicate that an init process should be used as
+the PID 1 in the container. Specifying an init process ensures the usual
+responsibilities of an init system, such as reaping zombie processes, are
+performed inside the created container.
+
+The default init process used is the first `docker-init` executable found in the
+system path of the Docker daemon process. This `docker-init` binary, included in
+the default installation, is backed by [tini](https://github.com/krallin/tini).
+
```

Comment:
For a follow-up; I _think_ the init-binary to use is configurable on the daemon (if so, we may need a link to that part of the daemon config)